### PR TITLE
bsunanda:Run2-alca8   Latest changes to meet calibration codes for HCAL

### PR DESCRIPTION
**Pull request by bsunanda**
Small changes to AlCa code; facility to look into HBHE muon calibration systematics and a few cfg files

### CODE REVIEW

**Comment by deguio** [comment]
+1
the DQM configs still use EDM format instead of DQMIO, but I don't see a problem

**Comment by mmusich** [comment]
@cmsbuild please test

File: Calibration/HcalCalibAlgos/test/gammaJetAnalysis_cfg.py [hunk 1]
```diff
@@ -1,13 +1,66 @@
 import FWCore.ParameterSet.Config as cms
 process = cms.Process('ANALYSIS')
 
-process.load('FWCore.MessageService.MessageLogger_cfi')
 process.load('Configuration.StandardSequences.Services_cff')
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```
Comment (diguida) [inline Calibration/HcalCalibAlgos/test/gammaJetAnalysis_cfg.py]: @bsunanda 
please change this to `process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')` in order to consume conditions from the new backend. The old one is decommissioned.

File: Calibration/HcalCalibAlgos/test/gammaJetAnalysis_cfg.py [hunk 2]
```diff
@@ -1,13 +1,66 @@
...
+        DEBUG = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        GammaJetAnalysis = cms.untracked.PSet(
+            limit = cms.untracked.int32(-1)
+       )
+    )
+)
 
 process.load("Configuration.Geometry.GeometryIdeal_cff")
```
Comment (diguida) [inline Calibration/HcalCalibAlgos/test/gammaJetAnalysis_cfg.py]: @bsunanda 
please change this to `process.load("Configuration.StandardSequences.GeometryRecoDB_cff") as explained [here](https://indico.cern.ch/event/368326/contribution/1/material/slides/0.pdf)

**Comment by diguida** [comment]
@bsunanda the changes are fine, but the ones in the tests.
I can approve, provided that you make this changes in another PR.

File: Calibration/HcalCalibAlgos/test/gammaJetAnalysis_cfg.py [hunk 3]
```diff
@@ -1,13 +1,66 @@
...
 process.load("MagneticField.Engine.autoMagneticFieldProducer_cfi")
```
Comment (diguida) [inline Calibration/HcalCalibAlgos/test/gammaJetAnalysis_cfg.py]: Also here, please move to `condDBv2`:

````
from Configuration.AlCa.autoCond_condDBv2 import autoCond
 process.GlobalTag.globaltag=autoCond['run2_mc']
````

**Comment by diguida** [comment]
+1
The code changes are satisfactory.
There are few issues related to consuming new condition DB backend. Not problematic, as they are only in tests, but to be fixed in another PR: we suggest @bsunanda to review all the test files and make one PR with all changes therein.

**Comment by davidlange6** [comment]
+1

**Pull request merged**